### DISCUSSION TITLE
SOMALI PROGRAMMING TEAM

### DISCUSSION THREAD

**Discussion by softwarepioneers** [community/community#81780] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

**MY VISION**
MAKING SOMALI PROGRAMMING TEAM
I AM FOUNDER IBRAHIM SICIID JAMA

**Comment by Akash1134** [comment, marked as answer]
Hi @softwarepioneers , we’re glad you’re here 😀

Thanks for sharing your (project/website/etc) with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
